metagraph-katana-plugin/metagraph_katana/plugins/katana_for_mg/translators.py

```diff
@@ -0,0 +1,79 @@
+import os
+# import sys
+
+# from executing.executing import NodeFinder
+import katana.local
+from katana.example_utils import get_input
+from katana.galois import set_active_threads
+# from pathlib import Path
+from icecream import ic
+import pandas as pd
+import pyarrow as pa
+import csv
+from scipy.sparse import csr_matrix
+
+
+import metagraph as mg
+from metagraph import translator
+from metagraph.plugins import has_networkx
+
+from .. import has_katana
+
+
+if has_katana and has_networkx:
+    import networkx as nx
+    from katana.property_graph import PropertyGraph
+    import numpy as np
+    from .types import KatanaGraph
+    from metagraph.plugins.python.types import dtype_casting
+    from metagraph.plugins.networkx.types import NetworkXGraph
+    from typing import Tuple
+    
+    @translator
+    def nx_to_kg(x:NetworkXGraph, **props) -> KatanaGraph:
```

Comment:
Is the name forced by the metagraph interface? If not, can we called this "networkx_to_katanagraph".